bundles/binding/org.openhab.binding.xbmc/src/main/java/org/openhab/binding/xbmc/rpc/RpcCall.java

```diff
@@ -131,11 +131,14 @@ private void postRequest(Map<String, Object> request, Runnable completeHandler)
         try {
             // we fire this request off asynchronously and let the completeHandler
             // process any response as necessary (can be null)
-            ListenableFuture<Response> future = client.preparePost(uri).setBody(writeJson(request))
+            String resultWrite = writeJson(request);
+			logger.debug("Write JSON: {}", resultWrite ); // Stefano: logging the JSON request
```

Comment:
Please put the code through the Eclipse IDE formatter so future changes don't reformat lines with incorrect formatting.